### PR TITLE
Webserver scp upload

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Creating webserver part of the scp upload

### CODE REVIEW

File: src/main/webserver/routes/upload-scp.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+router.post("/", multipartMiddleware, (req, res) => {
+    const newPath = `${__dirname}/../project-scp-fies/project.scp`;
+    if (req.files.project.name.endsWith(".scp")) {
+        fs.rename(req.files.project.path, newPath, (err) => {
+            if (err) {
+                res.json({
+                    succes: false, message: "Some error occurred, please try again later!" });
+            } else {
+                res.json({ succes: true, message: "Project successfully uploaden!" });
```
Comment (Markvanderuit) [inline src/main/webserver/routes/upload-scp.js:15]: spelling error, just thought I'd mention it.

**Comment by Markvanderuit** [comment]
Looks good. Waiting for checks and second member to confirm.

**Comment by mennoo1996** [comment]
Good work!

File: src/main/webserver/routes/upload-scp.js (hunk 1 above)
Comment (bart-de-jonge) [inline src/main/webserver/routes/upload-scp.js:15]: WIll fix this later

**Pull request merged**